docs/online-experiments/firebase.md

```diff
@@ -0,0 +1,111 @@
+```
+- Click on `Next`
+- You will not need to run the command that is prompted in after clicking next, click `Next` again
+- Click `Continue to console`
+
+### Setting up Firestore
+Autora includes cloud storage for task data using Firestore. Follow these steps to initialize Firestore:
+
+- Navigate to the current project in the developer console and select Firestore Database from the sidebar under build.
+- Click `Create Database`
+- Select production mode and click `Next`
+- choose the current location and click `Enable`
+
+### Configure your project for Firebase
+In the project folder, write the following commands in your terminal:
+First login to your firebase account using
+```shell
+firebase login
+```
+Then initialize the firebase project to this folder by running:
+```shell
+firebase init
+```
+An interactive init process will now run in your command line. For the first question, select these options:
+
+- Firestore: Configure security rules and indexes files for Firestore
+- Hosting: Configure files for Firebase Hosting and (optionally) set up GitHub Action deploys
+- As a Firebase project, use the one, you created earlier
+- Use the default options for the Firestore Rules and the Firestore indexes.
+- ***!!! IMPORTANT !!!*** Use the build directory instead of the public directory here.
+- When asked for the directory, write `build` and press `Enter`.
+- Configure as a single-page app, don't set up automatic builds and deploys with GitHub. 
+- Don't overwrite the index.html file if the question pops up.
+
+## Write code for your experiment
+To write code for your experiment, use the `main.js` file in the `src/design` folder. For example, you can use [jsPsych](https://www.jspsych.org/7.3/) and install packages using npm. The main function should return an observation (the data created by a participant).
+
+You can test the experiment locally using
+```shell
+npm start
+```
+During the development, the Firestore database will not be used. If you want to load conditions from the database, you need to upload them first (for example using the [AutoRA Firebase experimentation manager](user-guide/experiment-runners/experimentation-managers/firebase/)) and set `REACT_APP_devNoDb="False"` in the .env file.
+
+### Using Prolific Ids
+If you want to recruit participants via Prolific (for example using the [AutoRA Prolific participant manager](user-guide/experiment-runners/recruitment-managers/prolific/)), we ***highly recommend*** setting `REACT_APP_useProlificId="True"`. This will speed up the recruitment of participants.
+
+## Build and deploy the project to Firebase 
+To serve the website to the internet, you need to build and deploy it to Firebase.
+To build the project, run
+```shell
+npm run build
+```
+To deploy to Firebase, run
+```shell
+firebase deploy
+```
+This will expose the website to the web. You can find the url of the website in the command line or on the Firebase console of your project under `Hosting`.
```

Comment:
```suggestion
This will make the website available on the web. You can find the URL of the website in the command line or on the Firebase console of your project under `Hosting`.
```